app/views/content/become-a-further-education-teacher.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Become a further education teacher.
+article_classes: ['longform']
+image: "media/images/content/hero-images/0015.jpg"
+description: |-
```

Comment:
@Tomcowan1  I just used the opening sentence from the doc here. Feel free to replace the description with a better synopsis of the doc. This is what'll appear beneath the link when the page appears in Google results, like the grey text here:

![Screenshot from 2021-06-15 15-57-52](https://user-images.githubusercontent.com/128088/122076082-798d1180-cdf2-11eb-8fd6-52173ebb0bf3.png)